src/features/Auth/Login.jsx

```diff
@@ -12,9 +13,54 @@ import {
   Text,
   useColorModeValue,
 } from "@chakra-ui/react";
+import { loginUser } from "./authSlice";
+import { useNavigate } from "react-router-dom";
+import { useDispatch } from "react-redux";
 import { Link as ReactRouterLink } from "react-router-dom";
+import { toast } from "react-toastify";
 
 export function Login() {
+  const navigate = useNavigate();
+
+  const dispatch = useDispatch();
+
+  const [userDetails, setUserDetails] = useState({
+    email: "",
+    password: "",
+    isRememberMe: true,
+  });
+
+  const changeHandler = (e) => {
+    setUserDetails({ ...userDetails, [e.target.name]: e.target.value });
+  };
+
+  const loginHandler = (e) => {
+    e.preventDefault();
+    if (userDetails.email === "" || userDetails.password === "") {
+      toast.error("Please enter both email and password", {
+        position: "top-right",
```

Comment:
suggestion: You can define the props of react toastify in ```ToastContainer``` itself

Reply:
Thanks for telling I did not think of it. Will do that.